spyder/plugins/editor/widgets/codeeditor.py

```diff
@@ -630,14 +630,22 @@ def setup_editor(self,
                      occurrence_timeout=1500,
                      show_class_func_dropdown=False,
                      indent_guides=False,
-                     scroll_past_end=False):
+                     scroll_past_end=False, 
+                     debug_panel=True,
+                     folding=True):
 
         self.set_close_parentheses_enabled(close_parentheses)
         self.set_close_quotes_enabled(close_quotes)
         self.set_add_colons_enabled(add_colons)
         self.set_auto_unindent_enabled(auto_unindent)
         self.set_indent_chars(indent_chars)
 
+        # Hide debug panel depending on the language

```

Comment:
@juanis2112 please change this comment to something like `Show/hide the debug panel depending on the language and parameter`